core/src/streams/clients.rs

```diff
@@ -253,11 +253,23 @@ impl StreamsClients {
                     chunk,
                 );
 
+                let alias_name = config
+                    .events
+                    .iter()
+                    .find(|n| n.event_name == event_message.event_name)
+                    .and_then(|n| n.alias.clone());
+
+                let event_message = EventMessage {
+                    event_name: alias_name.unwrap_or(event_message.event_name.clone()),
```

Comment:
`alias_name.unwrap_or_else` tend to be standard clippy

Reply:
Yes, `unwrap_or_else` is what I want here 👍